streampark-common/src/test/scala/org/apache/streampark/common/util/DateUtilsTest.scala

```diff
@@ -0,0 +1,160 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.streampark.common.util
+
+import org.junit.jupiter.api.Assertions._
+import org.junit.jupiter.api.Test
+
+import java.text.SimpleDateFormat
+import java.time.{Instant, LocalDateTime}
+import java.util.{Date, TimeZone}
+import java.util.concurrent.TimeUnit
+
+class DateUtilsTest {
+  val dateTestCase = "2000-01-01 00:00:01"
+  val timeStampTestCase: Long = 946656001000L;
+
+  @Test def stringToDateTest(): Unit = {
+    val sdf: SimpleDateFormat = new SimpleDateFormat(DateUtils.fullFormat)
```

Comment:
You can try something else, For example, https://github.com/apache/incubator-streampark/tree/dev/streampark-flink/streampark-flink-kubernetes-v2/streampark-fli nk-kubernetes-core/src/test/scala/org/apache/streampark/flink/kubernetes/v2